manifest.json

```diff
@@ -90,6 +90,14 @@
     "nodeVersion": "6.0.0",
     "commands": ["ember-3x-codemods cp-property-map app/**/*.js"]
   },
+  "ember-angle-brackets-codemod": {
+    "versions": {
+      "ember-source": "3.10.0-alpha.1"
```

Comment:
This version doesn't exist. We should probably use 3.10.0-beta.1.

Reply:
Updated